analyzer/src/testFixtures/kotlin/TestUtils.kt

```diff
@@ -40,7 +40,7 @@ import org.ossreviewtoolkit.model.config.RepositoryConfiguration
 import org.ossreviewtoolkit.model.config.ScopeExclude
 import org.ossreviewtoolkit.model.config.ScopeExcludeReason
 import org.ossreviewtoolkit.plugins.api.PluginConfig
-import org.ossreviewtoolkit.utils.test.USER_DIR
+import org.ossreviewtoolkit.utils.common.getCommonParentFile
```

Comment:
commit-msg: Is there another reason for no more using `USER_DIR` which is not mentioned?

Reply:
Well, as a side effect it helps to make `test-utils` a *little* bit less Java-specific, which in turn could help to make it a KMP project.

Reply:
From below comments, is one of the motivations to make it work with resources?